src/main/resources/view/MainWindow.fxml

```diff
@@ -12,7 +12,7 @@
 <?import javafx.scene.layout.VBox?>
 <?import javafx.stage.Stage?>
 
-<fx:root minHeight="725" minWidth="450" onCloseRequest="#handleExit" title="Weme" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1">
+<fx:root minHeight="725" minWidth="450" onCloseRequest="#handleExit" title="Weme" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
why are we dropping version here?

Reply:
JavaFX is complaining when we use 11. Idk why.